src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  errorMessage: ErrorType;
+  onCloseError: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  onCloseError,
+}) => {
+  useEffect(() => {
+    const timeoutID = setTimeout(() => {
+      onCloseError();
+    }, 3000);
+
+    return () => {
+      clearTimeout(timeoutID);
+    };
+  }, []);
```

Comment:
missed dependency
https://react.dev/learn/removing-effect-dependencies#dependencies-should-match-the-code